mw-webapp/src/component/tag/Tag.tsx

```diff
@@ -47,25 +68,34 @@ interface TagProps {
  * WayCard tag component
  */
 export const Tag = (props: TagProps) => {
-  return (
-    <button className={styles.tagContainer}>
-      <span
-        className={clsx(styles.cross, !props.isDeletable && styles.hiddenCross)}
-        onClick={() => {
-          props.isDeletable && props?.onDelete && props.onDelete(props.tagName);
-        }}
-        data-cy={props.cy?.dataCyCross}
-      >
-        X
-      </span>
+  return props.type === TagType.BASIC_TAG ?
+    (
+      <button className={styles.tagContainer}>
```

Comment:
Let's use our component Button if we need tag button)

Reply:
Done